### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by tool4ever**

### CODE REVIEW

File: forge-gui/res/cardsfolder/e/eaten_by_spiders.txt [hunk 1]
```diff
@@ -1,7 +1,6 @@
 Name:Eaten by Spiders
 ManaCost:2 G
 Types:Instant
-A:SP$ Destroy | Cost$ 2 G | ValidTgts$ Creature.withFlying | TgtPrompt$ Select target creature with flying | RememberAttached$ True | SubAbility$ ChompEquip | SpellDescription$ Destroy target creature with flying and all Equipment attached to that creature.
-SVar:ChompEquip:DB$ DestroyAll | ValidCards$ Remembered.Equipment | SpellDescription$ Destroy all Equipment attached to that creature. | SubAbility$ Cleanup
-SVar:Cleanup:DB$ Cleanup | ClearRemembered$ True
+A:SP$ Pump | Cost$ 2 G | ValidTgts$ Creature.withFlying | TgtPrompt$ Select target creature with flying | SubAbility$ ChompEquip | SpellDescription$ Destroy target creature with flying and all Equipment attached to that creature.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/e/eaten_by_spiders.txt:4]: probably a StackDescription$ REP here and `None` on the next line and that's tidied up too?

Comment (Northmoc) [reply forge-gui/res/cardsfolder/e/eaten_by_spiders.txt]: Maybe this needs Curse too? Or is AI able to see that it leads to destroying the target?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/e/eaten_by_spiders.txt]: good point

File: forge-gui/res/cardsfolder/e/eaten_by_spiders.txt [hunk 2]
```diff
@@ -1,7 +1,6 @@
 Name:Eaten by Spiders
 ManaCost:2 G
 Types:Instant
-A:SP$ Destroy | Cost$ 2 G | ValidTgts$ Creature.withFlying | TgtPrompt$ Select target creature with flying | RememberAttached$ True | SubAbility$ ChompEquip | SpellDescription$ Destroy target creature with flying and all Equipment attached to that creature.
-SVar:ChompEquip:DB$ DestroyAll | ValidCards$ Remembered.Equipment | SpellDescription$ Destroy all Equipment attached to that creature. | SubAbility$ Cleanup
-SVar:Cleanup:DB$ Cleanup | ClearRemembered$ True
+A:SP$ Pump | Cost$ 2 G | ValidTgts$ Creature.withFlying | TgtPrompt$ Select target creature with flying | IsCurse$ True | SubAbility$ ChompEquip | StackDescription$ REP target_{c:Targeted} | SpellDescription$ Destroy target creature with flying and all Equipment attached to that creature.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/e/eaten_by_spiders.txt:4]: I think `REP target creature with flying_{c:Targeted}`

Sorry I didn't notice that before

Comment (tool4ever) [reply forge-gui/res/cardsfolder/e/eaten_by_spiders.txt:4]: nah that's on me for not checking again :)

**Review by Northmoc — APPROVED** [review]
👍

**Pull request merged**